en/micro-integrator/docs/develop/using-remote-micro-integrator.md

```diff
@@ -7,14 +7,18 @@ The following instructions can be used to run your artifacts in a remote Micro I
 ## Deploy and run artifacts in a remote instance
 
 1.	[Download and install](../../setup/installation/install_in_vm) the Micro Integrator server and on your computer. 
-2.	[Export your packaged artifacts](packaging-artifacts.md) from WSO2 Integration Studio.
-3.	Copy the exported CAR file to the `<MI_HOME>/repository/deployment/server/carbonapps/` folder where <MI_HOME> is the root folder of your Micro Integrator installation.
+2.	[Package your Synapse artifacts](packaging-artifacts.md) from WSO2 Integration Studio.
+3.	[Add a remote Micro Integrator or ESB Server](add-new-remote-server.md) and deploy the the artifacts.
```

Comment:
Since the pointed doc is about adding and deploying, we need to hyperlink the whole sentence here.